core/trino-main/src/main/java/io/trino/server/ui/FormUiAuthenticatorModule.java

```diff
@@ -47,6 +48,7 @@ public void configure(Binder binder)
             binder.bind(FormAuthenticator.class).to(InsecureFormAuthenticator.class).in(SINGLETON);
         }
         configBinder(binder).bindConfig(FormWebUiConfig.class);
```

Comment:
it seems that `PasswordAuthenticatorManager` should be binded only when `usePasswordManager=true`